ecosystem.config.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+module.exports = {
+	apps: [
+		{
+			name: 'lisk-core',
+			script: './bin/run',
+			args: 'start',
+			env: {
+				LISK_NETWORK: 'mainnet',
```

Comment:
I would also recommend to add all the supported environment variables with thier default values
```suggestion
				LISK_NETWORK: 'mainnet',
				LISK_API_IPC: true,
				LISK_ENABLE_FORGER_PLUGIN: true,
				LISK_API_WS: true,
				LISK_API_WS_PORT: 7887,
				LISK_CONSOLE_LOG_LEVEL:  'warn',				
```

Reply:
Applied